sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -1412,6 +1412,10 @@ def callback_mock(renewable, error):
                 errors.append(error)
 
         receiver = MockReceiver()
+        auto_lock_renew = AutoLockRenewer()
+        with pytest.raises(TypeError):
+            auto_lock_renew.register(Exception()) # an arbitrary invalid type.
```

Comment:
emmm, should it be like this?
```suggestion
            auto_lock_renew.register(receiver, Exception()) # an arbitrary invalid type.
```